dune-precice/dune.module

```diff
@@ -7,4 +7,4 @@ Module: dune-precice
 Version: 1.0
 Maintainer: max.firmbach@unibw.de
 #depending on
-Depends: dune-common (>= 2.7) dune-istl (>= 2.7) dune-functions (>= 2.7)
+Depends: dune-common (>= 2.9) dune-istl (>= 2.9) dune-functions (>= 2.9)
```

Comment:
This change is not relevant to preCICE v3, right? Lets do this directly on the main branch.